run-tests.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -eu
+
+lambda_function_name=$(aws cloudformation describe-stacks --stack-name ipv-cri-check-hmrc-smoke-tests --query "Stacks[0].Outputs[?OutputKey=='CanaryInvokerFunction'].OutputValue" --output text)
```

Comment:
Getting multiple canaries :+1: 

If we found we had more canaries than we wanted to run post merge :shrug:  we could always tag them and search for them that way